### PR TITLE
docs(carousel): decomopose Dynamic Slides

### LINKED ISSUE #3651: docs(carousel): decomopose Dynamic Slides
Decomopose `Dynamic Slides` to: 
- Dynamic Slides
- Enable/Disable indicator
- Disable Slide Looping
- Interval

related to #3232

### PR DESCRIPTION
**Pull request by artolshansky**
Decompose Dynamic Slides section to: Dynamic slides, Disable indicator, Disable slide looping, Interval

close #3651

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Needs to fix interval demo - changing slides freezes for more than chosen timeout

**Comment by YevheniiaMazur** [comment]
Discussed comment - need to recheck issue's reproducing

**Comment by YevheniiaMazur** [comment]
Please, fix timeout for changing slides
carouselinterval

**Comment by artolshansky** [comment]
Interval work incorrect without `[(activeSlide)]`

File: demo/src/app/components/+carousel/demos/disable-looping/disable-looping.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-carousel-disable-looping',
+  templateUrl: './disable-looping.html'
+})
+export class DemoCarouselDisableLoopingComponent {
+  slides: any[] = [
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/disable-looping/disable-looping.ts]: remove type annotation

File: demo/src/app/components/+carousel/demos/disable-looping/disable-looping.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    {image: 'assets/images/nature/8.jpg'},
+    {image: 'assets/images/nature/7.jpg'},
+    {image: 'assets/images/nature/6.jpg'}
+  ];
+  noWrapSlides: boolean = false;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/disable-looping/disable-looping.ts]: same

File: demo/src/app/components/+carousel/demos/dynamic/dynamic.ts [hunk 3]
```diff
@@ -5,11 +5,8 @@ import { Component } from '@angular/core';
   templateUrl: './dynamic.html'
 })
 export class DemoCarouselDynamicComponent {
-  myInterval: number = 1500;
   slides: any[] = [];
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/dynamic/dynamic.ts]: same

File: demo/src/app/components/+carousel/demos/interval/interval.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-carousel-interval',
+  templateUrl: './interval.html'
+})
+export class DemoCarouselIntervalComponent {
+  myInterval: number = 1500;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/interval/interval.ts]: :)

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**